opendatahub/quickstart/pvc/README.md

```diff
@@ -1,22 +1,12 @@
-# Quick Start Guide - PVC Feature
+# Quick Start - Sample Model Deployment and Persistent Volume Claim
```

Comment:
sync with the title in another doc.
~~~
# Quick Start - Sample Model Deployment using Persistent Volume Claim
~~~